src/module_compilation.ml

```diff
@@ -196,13 +201,13 @@ let build_module ~dep_graphs cctx m =
     SC.add_rules sctx ~dir
       (Js_of_ocaml_rules.build_cm cctx ~js_of_ocaml ~src ~target))
 
-let build_modules ~dep_graphs cctx =
+let build_modules ~dep_graphs ?precompiled_cmi cctx =
   Module.Name.Map.iter
     (match CC.alias_module cctx with
      | None -> CC.modules cctx
      | Some (m : Module.t) ->
        Module.Name.Map.remove (CC.modules cctx) (Module.name m))
-    ~f:(build_module cctx ~dep_graphs)
+    ~f:(build_module cctx ~dep_graphs ?precompiled_cmi)
```

Comment:
I believe `build_modules` shouldn't take this as a parameter and always pass `~precompiled_cmi:false`. But I got rid of this function in my PR anyway, so once that's merged this can be simply rebased.

Reply:
I suppose yh, I removed for now as it's unused. That'll make rebasing easier